exchange/exchange.go

```diff
@@ -316,8 +316,17 @@ func (e *exchange) HoldAuction(ctx context.Context, r *AuctionRequest, debugLog
 
 	recordImpMetrics(r.BidRequestWrapper, e.me)
 
+	// Retrieve host and account-level EEA countries config
+	eeaCountries := SelectEEACountries(e.privacyConfig.GDPR.EEACountries, r.Account.EEACountries)
+
+	// Create a map for efficient lookup
+	eeaCountriesMap := make(map[string]struct{})
```

Comment:
The performance benefit of a map only comes into play when the map is used more than once. As it is now, this map is only referenced once in `parseGDPRDefaultValue`. This forces the processing of every element with the overhead of map creation. It would be more efficient to use a linear search in this instance.

Eventually, this map can live in the account config object when that caching layer is improved. Doesn't make sense there right now though.